Dockerfile

```diff
@@ -43,11 +47,4 @@ COPY --from=BUILD /go/bin/* /bin/
 ADD startup.sh /
 ADD ceph.conf.template /
 
-# VOLUME [ "/run/docker/plugins" ]
```

Comment:
Please, leave the commented volumes as they are used for documentation purposes of which mount points have to be set

Reply:
@tiagostutz, please check what do you think about this.